brave/brave-site-specific-scripts
# Zendesk: do not attempt to send tracker for support.brave.com

Avoids shields block count

Depends on: https://github.com/brave/brave-site-specific-scripts/pull/7

## Test plan
- Browse support.brave.com
- Observe: shields blocks something
- `npm run build`
- Copy dist/ to Brave user profile dir (greaselion extension sub-dir)
- Restart Brave
- Browse support.brave.com
- Observe: shields does not block anything
